### PR TITLE
tests: check exception policy stats counters - v8

### PR DESCRIPTION
**Pull request by jufajardini**
Edit the existing exception policy tests to check for the new exception policy stats counters.

Ticket #5816

Previous PR: https://github.com/OISF/suricata-verify/pull/1746

Changes from previous PR:
- add a test to showcase the wider column for counters' names in stats.log

## Ticket

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/5816

### CODE REVIEW

**Comment by jufajardini** [comment]
forced-puch to remove duplicated pcap for `exception-policy-applayer-03` test

**Comment by jufajardini** [comment]
Suricata PR: https://github.com/OISF/suricata/pull/10785

File: tests/exception-policy-applayer-01/suricata.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 %YAML 1.1
 ---
 
+stats:
+  enabled: yes
+  interval: 8
```
Comment (jufajardini) [inline tests/exception-policy-applayer-01/suricata.yaml:6]: Wondering if this should be a shorter interval, so we could possibly see stats records with and without the exception policy stats incremented.

Comment (victorjulien) [reply tests/exception-policy-applayer-01/suricata.yaml:6]: I think there are some implementation issues with reducing it, so lets not worry about it now

**Comment by victorjulien** [comment]
Merged in #1752, thanks!

**Pull request closed without merging**